src/redux/flows/flows.reducer.ts

```diff
@@ -175,3 +226,27 @@ function _loadFlow(state: FlowState, flowId: string): FlowState {
     selectedFlow: flowId,
   };
 }
+
+function _updateNode(
+  state: FlowState,
+  { nodeId, nodeUpdate }: { nodeId: string; nodeUpdate: NodeUpdate }
+): FlowState {
+  const newState = { ...state };
```

Comment:
Cloning with spread operator has a problem. It makes a shallow copy. https://www.javascripttutorial.net/object/3-ways-to-copy-objects-in-javascript/

This could cause bugs with child properties being updated in previous state. I'll look into suggestions like immer from redux.

Reply:
https://stevenmercatante.com/simplify-redux-reducers-with-immer

Reply:
Got you, it makes the most sense though to implement immer overall (like also pointed out in the tutorial), which should be done in a separate PR